src/utils/use-title.ts

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+
+const setTitle = (title: string) => {
+  document.title = title
+}
+
+export const useTitle = (customString?: string) => {
+  const urlParams = useParams()
+  useEffect(() => {
+    const currentTitle = document.title
+    let pageTitle = `Lora`
```

Comment:
I think it would be good for all pages to start with `Lora - ${pageTitle}`
So:
`Lora - Transaction Wizard`
`Lora - TxnId:CFNKY4JV5BMIRSP2SMZE634DYB22FALLURI72U6UMWZNKEBOYEMQ - mainnet`

Probably the easiest way to do this is accumulate in an array, then `.join(' - ')` to build the page title. Something like below:

```
  const params = new Array<string>()

  if (urlParams.transactionId) {
    params.push(`TxnId:${urlParams.transactionId}`)
  }
  // ...

  const title = `Lora${pagePrefix ? ` ${pagePrefix}` : ''} - ${params.join(' - ')}`
```

Reply:
I have a small preference for omitting the `Id` part in transactions / assets / apps

So: `Txn:${id}` `App:${id}` `Asset:${id}` 

What do you think?

Reply:
And if we omit the `-` separators, browser tabs would show more of the title before truncating:

`Lora Txn ACSASDSADASD Mainnet` vs `Lora - Txn:ACSASDSADASD - Mainnet`

May not make much difference though, so this is quite subjective

Reply:
I'm happy with both of those suggestions.

Reply:
Me too. I'll make these updates today.